db/compaction/compaction_job.cc

```diff
@@ -713,9 +718,13 @@ Status CompactionJob::Install(const MutableCFOptions& mutable_cf_options) {
   cfd->internal_stats()->AddCompactionStats(
       compact_->compaction->output_level(), thread_pri_, compaction_stats_);
 
+  versions_->SetIOStatusOK();
```

Comment:
Could this be called once in DBImpl::ResumeImpl() instead of at the start of every compaction/flush?